scripts/dashboard-importer/README.md

```diff
@@ -0,0 +1,185 @@
+# Cloud Monitoring Dashboard Importer
+
+Convert dashboards from the Grafana dashboard JSON schema to the Cloud
+Monitoring dashboard JSON schema and upload them directly into Cloud Monitoring.
+
+## Prerequisites
+To run the dashboard importer, you must have both Node.js and gcloud installed.
+1. Node.js
+    - [Installation Docs](https://nodejs.org/en/download)
+    - Recommended min Node.js version: 20.4.1
+1. gcloud
+    - [Installation Docs](https://cloud.google.com/sdk/docs/install)
+    - [Ensure your gcloud version is up to date](https://cloud.google.com/sdk/gcloud/reference/components/update)
+    - [Obtain access credentials](https://cloud.google.com/sdk/gcloud/reference/auth/login) by running `gcloud auth login`
+
+## Set Up
+Clone the `monitoring-dashboard-samples` repo and CD into the `dashboard-importer` directory
+```
+git clone https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples.git && cd dashboard-importer
+```
+
+## Quick Start
+### Converting and Uploading Dashboards
+To convert Grafana dashboard definitions into the Cloud Monitoring format and
+upload them to your Google Cloud project, run the following:
+
+```
+./import.sh <FILE_OR_DIRECTORY> <PROJECT_ID>
+```
+
+#### Arguments:
+  - FILE_OR_DIRECTORY: Path to a file or directory that contains Grafana dashboards in JSON format. These files must use the extension `.json`.
+  - PROJECT_ID: The destination Google Cloud project for the converted dashboards
+
+#### Example:
+  To convert a single file `./foo/test.json` and upload to project `test-project`, you run:
+  ```
+  ./import.sh ./foo/test.json test-project
+  ```
+
+  To convert all the JSONs in the `./foo` directory and upload to project `test-project`, you run:
+  ```
+  ./import.sh ./foo/ test-project
+  ```
+
+#### Behavior and Output:
+The importer will convert the JSON files in the given path and upload them to
+the specified project using gcloud.
+
+The importer will also generate some files. To view your converted JSON files, look under the `reports/<date>/<time>/` directory the importer.
```

Comment:
"directory the importer"

"directory in the importer's directory"?

Reply:
Updated!